website/docs/quick-start/installation/modal/index.md

```diff
@@ -40,22 +49,37 @@ def download_model():
 
     subprocess.run(
         [
-            "/opt/tabby/bin/tabby-cpu",
+            TABBY_BIN,
             "download",
             "--model",
             MODEL_ID,
-        ]
+        ],
+        env=TABBY_ENV,
     )
-```
 
 
+def download_chat_model():
+    import subprocess
+
+    subprocess.run(
+        [
+            TABBY_BIN,
+            "download",
+            "--model",
+            CHAT_MODEL_ID,
+        ],
+        env=TABBY_ENV,
+    )
+
+```
+
```

Comment:
also need the embedding model download?

Reply:
Yes, I've increased the content of the cached embedding model.